docs/index.rst

```diff
@@ -2015,8 +2042,19 @@ Full Default Bundle Configuration
                     'object_managers' => false,
                 ],
 
-                // Add global state
-                'global_state' => []
```

Comment:
there was an error in the doc: `global_state` should be a direct child of `zenstruck_foundry`